Documentation updates (#2450)
openturns/openturns: python/doc/theory/reliability_sensitivity/taylor_moments.rst

    \Cov \uX = \mat{C}

where :math:`\mat{C} \in \Rset^{n_X \times n_X}` is the input covariance matrix:

.. math::

    c_{ij} = \Expect{\left(X_i - \Expect{X_i}\right)\left(X_j - \Expect{X_j} \right)}

for :math:`i, j \in \{1, ..., n_X\}`.

> Notice that each diagonal element of the covariance matrix :math:`c_{ii} = \sigma_i^2`, is

Reviewer: ```suggestion
Notice that each diagonal element of the covariance matrix :math:`c_{ii} = \sigma_i^2` is
```

> Notice that the diagonal of the covariance matrix, :math:`c_{ii} = \sigma_i^2`, is
> equal to the variance of the input variable :math:`X_i`.

Reviewer: ```suggestion
Notice that each diagonal element of the covariance matrix :math:`c_{ii} = \sigma_i^2`, is
equal to the variance of an input variable (:math:`X_i`).
```

Case 1: scalar output
~~~~~~~~~~~~~~~~~~~~~

In this section, we analyse the special case where :math:`q = 1` and :math:`Y = h(\vect{X}) \in \Rset`.
The second-order Taylor expansion of :math:`h` at the point :math:`\ux = \vect{\mu}` is:

.. math::

    h(\ux) = h(\vect{\mu}) + \sum_{i = 1}^d \frac{\partial h}{\partial x_i} (\vect{\mu})(x_i - \mu_i)
    + \frac{1}{2} \sum_{i,j = 1}^d \frac{\partial^2 h}{\partial x_i \partial x_j}(\vect{\mu})
    (x_i - \mu_i) (x_j - \mu_j) + o\left(\|\vect{x} - \vect{\mu}\|^2\right)

when :math:`\vect{x} \rightarrow \vect{\mu}`.
The expectation and variance of the first-order expansion are:

.. math::

    \Expect{Y} \approx h(\vect{\mu})

.. math::

    \Var{Y} \approx \sum_{i=1}^{d} \sum_{j=1}^{d}
        c_{ij} \frac{\partial h}{\partial x_i}(\vect{\mu}) \frac{\partial h}{\partial x_j}(\vect{\mu}).

> In the special case where the input are independent, then
> the covariance expressions simplifies and we get:

Reviewer: ```suggestion
In the special case where the inputs are independent, then
the variance expression is simplified and we get:
```
